### PR TITLE
Add build job for openshift/windows_exporter

### PR DESCRIPTION
**Pull request by mansikulkarni96**
Add a build job for openshift/windows_exporter 

Ran commands:
```
make update
```

### CODE REVIEW

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

File: ci-operator/config/openshift/prometheus-community-windows_exporter/openshift-prometheus-community-windows_exporter-master.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.19-openshift-4.14:
```
Comment (aravindhp) [inline ci-operator/config/openshift/prometheus-community-windows_exporter/openshift-prometheus-community-windows_exporter-master.yaml:6]: Don't we want to use rhel-9 and golang-1.20?

File: ci-operator/config/openshift/prometheus-community-windows_exporter/openshift-prometheus-community-windows_exporter-master.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: build
+  commands: |
+    wget https://github.com/prometheus/promu/releases/download/v0.14.0/promu-0.14.0.linux-amd64.tar.gz
```
Comment (aravindhp) [inline ci-operator/config/openshift/prometheus-community-windows_exporter/openshift-prometheus-community-windows_exporter-master.yaml:25]: Shouldn't we use the promu in the WMCO submodule?

**Review by aravindhp — COMMENTED** [review]
Thanks for doing this, @mansikulkarni96. I have a couple of questions.

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

File: core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+tide:
+  queries:
+  - labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
```
Comment (aravindhp) [inline core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml:8]: Do we need this?

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for working on this, @mansikulkarni96. Please address my comments.

File: core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml (hunk 3 above)
Comment (mansikulkarni96) [reply core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml:8]: I added it from what we have in the WMCO prowconfig. I don't see a usecase so will update it.

**Comment by mansikulkarni96** [comment]
/pj-rehearse pull-ci-openshift-prometheus-community-windows_exporter-master-build

File: core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml (hunk 3 above)
Comment (mansikulkarni96) [reply core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml]: I tried to adjust this but the bot is not happy due to org wide restrictions:
`the following orgs or repos enable the bugzilla plugin but do not forbid the bugzilla/invalid-bug label for merging: [repo: openshift/prometheus-community-windows_exporter`
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/38362/pull-ci-openshift-release-master-config/1653797247189520384/build-log.txt

File: core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+tide:
```
Comment (aravindhp) [inline core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml:1]: Don't you also need a `_pluginconfig.yaml` with:
```yaml
approve:
- commandHelpLink: https://go.k8s.io/bot-commands
  repos:
  - openshift/prometheus-community-windows_exporter
  require_self_approval: false
plugins:
  openshift/prometheus-community-windows_exporter:
    plugins:
    - approve
```
? Or is that not needed?

File: core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml (hunk 3 above)
Comment (aravindhp) [reply core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml]: Why don't we disable the bugzilla plugin in `_pluginconfig.yaml`?

File: core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml (hunk 4 above)
Comment (mansikulkarni96) [reply core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml:1]: @aravindhp _pluginconfig.yaml is present in this commit-> https://github.com/openshift/release/blob/5c69c25159d11e9b9078c7a3697171ff675afbea/core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_pluginconfig.yaml

Comment (aravindhp) [reply core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_prowconfig.yaml:1]: Oops, sorry I missed that.

File: core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_pluginconfig.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/prometheus-community-windows_exporter
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/prometheus-community-windows_exporter
+  review_acts_as_lgtm: true
```
Comment (aravindhp) [inline core-services/prow/02_config/openshift/prometheus-community-windows_exporter/_pluginconfig.yaml:9]: Do you want the `/approve` to imply `/lgtm`?

**Review by aravindhp — APPROVED** [review]

**Pull request merged**